### PR TITLE
fix: #1690 unable to identify shebang when it has initial spaces

### PR DESCRIPTION
**Pull request by yihong0618**
fix: #1690 
Signed-off-by:  yihong0618 <zouzou0208@gmail.com>

### CODE REVIEW

File: cmds/record.c [hunk 1]
```diff
@@ -19,6 +19,7 @@
 #include <sys/wait.h>
 #include <unistd.h>
 
+#include "libtraceevent/event-utils.h"
```
Comment (namhyung) [inline cmds/record.c]: I don't want to include libtraceevent headers just because of this.  Maybe we can implement `strim()` instead.

**Review by namhyung — COMMENTED** [review]
Can you please squash the commits into one and rewrite the commit message like this?

```
record: skip whitespaces after shebang for scripts

Python tracing won't work when the shebang line has a space like below:

    #! /usr/bin/python3

Fixed: #1690 

Signed-off-by: ...
```

**Comment by yihong0618** [comment]
> Can you please squash the commits into one and rewrite the commit message like this?
> 
> ```
> record: skip whitespaces after shebang for scripts
> 
> Python tracing won't work when the shebang line has a space like below:
> 
>     #! /usr/bin/python3
> 
> Fixed: #1690 
> 
> Signed-off-by: ...
> ```

of course

File: cmds/record.c (hunk 1 above)
Comment (yihong0618) [reply cmds/record.c]: will do this tonight

**Comment by namhyung** [comment]
Next time, you can reuse this PR instead of creating a new one.  You don't need to create a new PR whenever you update the code.  Just force-push is fine.

**Comment by yihong0618** [comment]
copy that, thanks

**Pull request closed without merging**